Gems/EMotionFX/Code/EMotionFX/Source/AnimGraphReferenceNode.cpp

```diff
@@ -644,6 +644,7 @@ namespace EMotionFX
     {
         ReleaseAnimGraphInstances();
 
+        AnimGraphNotificationBus::Broadcast(&AnimGraphNotificationBus::Events::OnReferenceAnimGraphAboutToBeChanged, this);
```

Comment:
Shouldn't this be called before ReleaseAnimGraphInstances()?
Look at the OnAssetReady() and OnAssetReload(), it seems like the AboutToBeChanged is fired before.

Reply:
Good call, changed.